meta-facebook/yv4-sd/src/platform/plat_pldm_monitor.c

```diff
@@ -0,0 +1,251 @@
+/*
```

Comment:
In general, i feel this code can again be moved into common layer rather than being platform specific?

Reply:
most of them has already been moved by PLDM set/get gpio change (#1307)